test/tests/slider_slider-color-viewer.js

```diff
@@ -423,14 +498,16 @@ ariaTest(
   exampleFile,
   'key-end',
   async (t) => {
+    sendAllSlidersToBeginning(t);
+
     const sliders = await t.context.queryElements(t, ex.sliderSelector);
 
     // Send key end to red slider
     const redSlider = sliders[0];
     await redSlider.sendKeys(Key.END);
 
     t.is(
-      await redSlider.getAttribute('aria-valuenow'),
+      await await redSlider.getAttribute('aria-valuenow'),
```

Comment:
```suggestion
      await redSlider.getAttribute('aria-valuenow'),
```

Reply:
@nschonni 
Had to put in two "awaits", otherwise the test intermittently failed.  Not sure why, but that fixed it for me at least.  I believe Valarie has used this technique on some other tests, that's why I tired it.

Reply:
Fair enough 😄